pandas/io/formats/format.py

```diff
@@ -1514,18 +1514,25 @@ def __init__(
 
     def _format_strings(self) -> List[str]:
         """ we by definition have DO NOT have a TZ """
-        values = self.values
-
-        if not isinstance(values, DatetimeIndex):
-            values = DatetimeIndex(values)
+        values = np.asarray(self.values)
+        flat_values = values.ravel() if values.ndim > 1 else values
```

Comment:
can we use the ```@ravel_compat``` here?

Reply:
Yeah, I think that would work. The only thing is that if `self.formatter` is set, then it would add an unnecessary conversion to an `numpy.ndarray ` and then back again to `list`. I'm not sure how crucial that case is, would you like me to go ahead and change it anyway?